deploy/crds/app.kiegroup.org_v1alpha1_kogitoexplainability_cr.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoExplainability
+metadata:
+  name: explainability
+spec:
+  # environment variables to set in the runtime container. Example: JAVAOPTS: "-Dquarkus.log.level=DEBUG"
+  # env: {}
+  # number of pods to be deployed
+  replicas: 1
+  # httpPort: 8080
+  # details about the kafka connection
+  kafka:
+    # let's leave this burden to KogitoInfra CR to deploy a new Kafka instance for us
+    useKogitoInfra: false
+  # details about the connected infinispan
```

Comment:
Explainability service doesn't use Infinispan, right?
This line can be removed then,

Reply:
Yep, thanks for the catch, you're right! Removed